src/app/event-detail/event-detail.component.html

```diff
@@ -6,9 +6,15 @@
     <span class="header-text">Explore Events</span>
   </div>
   <div *ngIf="event && event.properties" class="event-body">
-    <img class="event-image" src="{{event.properties.cover_picture}}">
+    <img class="event-image" src="{{event.properties.cover_picture == '<NONE>' ? '/assets/images/fb-banner-'+event.properties.categories[0].toLowerCase()+'.png' : event.properties.cover_picture}}">
```

Comment:
@dhs766 correct me if I'm wrong, but we don't have a 1-1 mapping from event category to banner. I think we grouped some categories together into a single banner. We might have to add a static mapping of categories -> banner image name. Additionally, we should default to some banner in the case that there is no banner image corresponding to the given category (e.g. the food category atm)